tests/performance/sync_test.go

```diff
@@ -0,0 +1,41 @@
+//go:build performance
+
+package performance
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Sync_Execution_Duration_Simple(t *testing.T) {
+	tests := []struct {
+		name                        string
+		stateFile                   string
+		acceptableExecutionDuration int64
+	}{
+		{
+			name: "Entities with UUIDs",
+			// This file contains 100 services, 10 consumer groups, and 100 consumers in total.
+			// Note that real world latency for http response will be about 10x of local instance (which is used in testing)
```

Comment:
I want to avoid unnecessary consumption of github actions usage by automated tests.